R/logit_utils.R

```diff
@@ -0,0 +1,83 @@
+#' @title Logit and inverse logit functions
+#'
+#' @description Logit and inverse logit functions, with options to scale input
+#'   to logit, scale output of inverse logit, or perform a transformation
+#'   such that the domain in non-logit space is as specified (standard
+#'   is (0, 1)).
+#'
+#' @param x \[`numeric()`\]\cr Value to take the logit or inverse logit of.
+#' @param scale \[`numeric()`\]\cr Value to scale by. Default 1.
+#' @param domain_lower \[`numeric()`\]\cr Lower bound of domain in non-logit
+#'   space, inclusive. Default 0. Logit at lower bound is -Inf.
+#' @param domain_upper \[`numeric()`\]\cr Upper bound of domain in non-logit
+#'   space, inclusive. Default 1. Logit at upper bound is Inf.
+#'
+#' @return \[`numeric()`\]\cr The calculated logit or inverse logit value.
+#'
+#' @details
+#' Standard:
+#'
+#' * logit (x) = log (x / (1-x))
+#' * inverse logit (x) = exp (x) / (1 + exp (x))
+#'
+#' Optional scaled logit with 'scale' argument:
+#'
+#' * logit (x, s) = log (s * x / (1 - s * x))
```

Comment:
I think this would clearer to use the same notation as the transformed domain section. Took me a second to realize this

```
* logit (x, s) = log (x' / (1 - x')) where x' = x * s
* inverse logit (x, s) = exp (x') / (1 + exp (x')) where x' = x * s
```